### ISSUE TITLE
Testing trained model

### ISSUE THREAD

**Issue by benliu961** [official-stockfish/nnue-pytorch#243]
Hello,

I have a trained model. How would I get predictions and testing accuracy using a test dataset?

Thank you!

**Comment by vondele** [comment]
That would be the validation loss, but generally we test newly trained nets by playing games with [run_games.py](https://github.com/glinscott/nnue-pytorch/blob/master/run_games.py). easy_train.py shows how this can be done. Nets that test well locally can be tested on fishtest as well.

**Comment by benliu961** [comment]
If I fine tuned the model so that it doesn't predict board evaluations anymore, how would I test it?

**Comment by Blimpyway** [comment]
What does it predict?

**Comment by benliu961** [comment]
It predicts a value that I have defined as Value of Computation. Its just some value between 0 and 1

**Comment by benliu961** [comment]
Is there no way to test it via tensorflow?

**Comment by Sopel97** [comment]
best you can do is look at loss
